package/src/languageFeatures.ts

```diff
@@ -649,20 +626,17 @@ export class ColorizationAdapter {
                     this.decorations = model.deltaDecorations(oldDecorations, newDecorations);
                 }
             })
-            .then(undefined, (err) => {
-                console.error(err);
+            .catch((err) => {
+                // Hack to avoid crashing calling code, while still logging the
+                // error. Might be better to just let the error propagate
+                // normally, but that would require more investigation.
+                setTimeout(() => {
+                    throw err;
```

Comment:
Why not just console.error? Or does that not handle the trace as well?

Reply:
That's right, console.error does not end up in our telemetry